### DISCUSSION TITLE
用了几天遇到的一些小问题探讨一下

### DISCUSSION THREAD

**Discussion by jh0072110** [Openwrt-Passwall/openwrt-passwall2#18] [Q&A]
我也不知道是不是我设置的问题（感觉应该不是吧）
国外部分一直蛮流畅没遇到什么问题…
问题就出在分流的国内部分
有些国内流媒体（什么爱什么艺之类的，放着放着就卡住了）
网易新闻的视频和图片间歇性的刷不出来
头条的视频也会间歇性的刷不出来…
我觉得不是些需要提出issue的问题…从这里探讨一下…看是不是有什么独到的体位哈哈哈哈……
PS：加油啊肉鸡大佬～

**Comment by jh0072110** [comment]
又尝试了一下 总之感觉还是有些许离谱的 不知道各位大佬们是不是不用国内服务 还是…我设置的有所问题……
<img width="582" alt="image" src="https://user-images.githubusercontent.com/50496854/160008337-9193fd8e-1ec9-440e-ac11-e49d562f33f4.png">
<img width="532" alt="image" src="https://user-images.githubusercontent.com/50496854/160008378-4b43a97f-4dbc-4d80-b8de-5a2e734097c9.png">
附上我个人设置的相关截图给大佬们探讨一下？

**Reply by handsomeji** [reply]
更新最新源码不保留配置再试下？

**Reply by jh0072110** [reply]
> 更新最新源码不保留配置再试下？

好的大佬

**Reply by jh0072110** [reply]
> 更新最新源码不保留配置再试下？

暂时好像没出什么问题，让我压力测试一阵～

**Reply by jh0072110** [reply]
![image](https://user-images.githubusercontent.com/50496854/160358533-71bebead-d098-4ac8-be16-097c772ef16b.png)
嗯 看起来问题还是存在…随便打开个视频就这样……

**Comment by yangfang043** [comment]
我是觉得不大稳定，  时间长了， 国内国外都打不开。 DNS也不能设置为AdGuard的端口

**Comment by handsomeji** [comment]
可能和设备性能有关系吧，我用x64没什么感觉

**Reply by jh0072110** [reply]
我j4125啊……

**Comment by jh0072110** [comment]
神奇的变好了，看起来是跟我去广告插件（adbyby+）有冲突（吧）=。=
